packages/wallets/polkadot.js/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# @swapkit/wallet-polkadot.js
```

Comment:
@anthon-dev isnt that just copy pasta from another wallet?

Reply:
Yes